internal/impl/devbox.go

```diff
@@ -852,6 +859,35 @@ func (d *Devbox) nixFlakesFilePath() string {
 	return filepath.Join(d.projectDir, ".devbox/gen/flake/flake.nix")
 }
 
+func (d *Devbox) getConfigEnvs(computedEnv []string) []string {
+	// convert key=value strings to map of {key: value}
+	computedEnvMap := map[string]string{}
+	for _, kvpair := range computedEnv {
+		splitKV := strings.SplitN(kvpair, "=", 2)
```

Comment:
if we move it into `computeNixEnv`, we can also avoid this silly splitting, but rather just iterate through the `map` before it gets, uhm, "pair-ified".

(We might have to refactor the function signature of the plugin envs though, which I just avoided because I was lazy).

Reply:
Btw, I did the refactor here: https://github.com/jetpack-io/devbox/pull/621